integration/helpers/setup.sh

```diff
@@ -40,11 +40,13 @@ EOF
 }
 
 # Start elasticsearch, if an S3 endpoint argument is passed in then enable it
+#
```

Comment:
Is this section still being used?

Reply:
This is showing examples of how to call the function